holoviews/operation/datashader.py

```diff
@@ -473,7 +473,13 @@ def _process(self, element, key=None):
                         x_range=x_range, y_range=y_range)
 
         dfdata = PandasInterface.as_dframe(data)
-        agg = getattr(cvs, glyph)(dfdata, x.name, y.name, agg_fn)
+        # Suppress numpy warning emitted by datashader
```

Comment:
Is there a corresponding issue/PR to fix this warning at the datashader level? Would be good to have a link to point to so this can be removed once fixed in datashader.

Reply:
I will open an issue in datashader and link it in this comment.

Reply:
There was actually an issue in datashader which I've linked in the comment.